rep-0003.rst

```diff
@@ -281,6 +281,68 @@ Build System Support:
 
 - Same as Indigo
 
+Melodic Morenia (May 2018 - May 2023)
+--------------------------------------
+Required Support for:
+
+- Ubuntu Artful (17.10)
+- Ubuntu Bionic (18.04)
+
+Recommended Support for:
+
+- Debian Stretch
+- Fedora 28
```

Comment:
Should we also target [fedora 27](https://fedoraproject.org/wiki/Releases/27/Schedule) ?

Reply:
I'd suggest not. With their [13 month support cycle](https://fedoraproject.org/wiki/Fedora_Release_Life_Cycle). If we release in May, we'll only get ~6 months of support.

Reply:
That's accurate.
Following that logic, should we also drop artful from the list?
With the Ubuntu 9-month release cycle, if we release end of may we'll only get ~2 months of support.

Reply:
It's certainly something we could consider. Ubuntu being our main development distro is a little different though. Artful will likely be what we do any testing on in the intermediate time.

Reply:
I agree with @tfoote about not support F27; the lifecycle is too short.  I would be in favor of dropping artful for the same reasons, but I'd like to get opinions from the community on that.

Reply:
I think historically the goal was to target all distros and not skip one, but it seems just unnecessary effort to support distros that we don't support for more that a few months. For Lunar we dropped Yakkety before even having the most used packages released. I would be in favor of dropping both F27 and Ubuntu 17.10

Reply:
+1 to skip artful, no one will use it anyway

Reply:
There is a standing document about the goals and the rational (see http://wiki.ros.org/Distributions/ReleasePolicy). Until that is revised and agreed upon we should stick to it which implies supporting Artful for the time frame it is available.

Reply:
We discussed this, and unless there is a strong reason to not support artful, we are going to support it to conform to the Release Policy.  As @tfoote suggests above, most of our development will probably be on artful anyway.